### PR TITLE
Adding a trial js file

### PR DESCRIPTION
**Pull request by sid-s1**

### CODE REVIEW

**Review by davebdev — APPROVED** [review]
Looks good Sid! Happy to merge

File: trial.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('will this work')
```
Comment (Chrispy1987) [inline trial.js:1]: Change console.log to 'Dave likes potato'

**Review by Chrispy1987 — CHANGES_REQUESTED** [review]
Hey lets be more specific about Daves love of potatoes

File: trial.js (hunk 1 above)
Comment (sid-s1) [reply trial.js:1]: No I don't think this can be allowed

**Review by Chrispy1987 — APPROVED** [review]

**Pull request merged**